duckduckgo/iOS
# sync save pdf

Task/Issue URL: https://app.asana.com/0/1199230911884351/1204383167189826/f
Tech Design URL:
CC:

**Description**:

Uses shared save PDF code and changes flow to share PDF from the settings screen

Note: Use BSK from https://github.com/duckduckgo/BrowserServicesKit/pull/320

**Steps to test this PR**:
1. Launch the app
2. Connect to sync if necessary
3. Tap on Save PDF button
4. Share sheet should appear
5. Save PDF somewhere accessible 
6. Check file contains expected output



**Copy Testing**:


**Orientation Testing**:


**Device Testing**:


**OS Testing**:


**Theme Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
